src/sql/workbench/contrib/resourceDeployment/browser/resourceDeployment.contribution.ts

```diff
@@ -15,3 +19,13 @@ CommandsRegistry.registerCommand({
 		return commandService.executeCommand('azdata.resource.deploy');
 	}
 });
+
+MenuRegistry.appendMenuItem(MenuId.ViewContainerTitle, {
+	group: 'deployment',
+	order: 4,
+	command: {
+		id: 'azdata.resource.deploy',
+		title: localize('deployment.title', "New Deployment...")
+	},
+	when: ContextKeyEqualsExpr.create('viewContainer', VIEWLET_ID)
```

Comment:
Does this also have it show up when the connections view pane is the only one there? That seems to be the current behavior so if we can keep that that'd be great

Reply:
Yes, the viewlet ID is the connection viewlet ID